src/ray/common/memory_monitor.cc

```diff
@@ -116,9 +116,19 @@ std::tuple<int64_t, int64_t> MemoryMonitor::GetCGroupMemoryBytes() {
 
   RAY_CHECK((total_bytes == kNull && used_bytes == kNull) ||
             (total_bytes != kNull && used_bytes != kNull));
+  if (used_bytes < 0) {
+    RAY_LOG_EVERY_MS(WARNING, kLogIntervalMs)
+        << "Got negative used memory for cgroup " << used_bytes << ", setting it to zero";
+    used_bytes = 0;
+  }
   if (total_bytes != kNull) {
-    RAY_CHECK_GT(used_bytes, 0);
-    RAY_CHECK_GT(total_bytes, used_bytes);
+    if (used_bytes >= total_bytes) {
```

Comment:
Should we set used_bytes to total_bytes in this case too?

Reply:
Sure